### PR TITLE
Test master & release workflow on PR merge

### PR DESCRIPTION
**Pull request by danielrbradley**
- Use a merge queue to ensure XYZ is able to be released with the latest ci-mgmt changes before the change is committed into master to be rolled out to all other providers.
- The next step here would be to auto-tag the release and verify that also completes correctly.

### CODE REVIEW

File: provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/release.yml [hunk 1]
```diff
@@ -1,4 +1,13 @@
 # WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt
+name: release
+on:
+  push:
+    tags:
+    - v*.*.*
+    - "!v*.*.*-**"
+#{{- if .Config.testMasterAndReleaseWorkflows }}#
+  pull_request:
+#{{ end }}#
```
Comment (iwahbe) [inline provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/release.yml:10]: Is this just a move?

**Review by iwahbe — APPROVED** [review]
This looks ok. LGTM

File: provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/release.yml (hunk 1 above)
Comment (danielrbradley) [reply provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/release.yml:10]: Yup, just had to change something to be able to open a PR downstream

**Review by flostadler — APPROVED** [review]
lgtm

**Pull request merged**